### DISCUSSION TITLE
Extra screen for "todays planned assignments"

### DISCUSSION THREAD

**Discussion by LucBen93** [AppStractSE/PropertyManager#86] [Ideas]
An extra screen where the admin can insert assignments for today that the property managers to see and interact with just as their regular customer/assignment view.

**Comment by LucBen93** [comment]
The risk with this is that the admins and property managers screens can become bloated? This will be one extra navigation and screen for both the admin and property manager.
Also will the admin have the time to micro-manage each team like that? 

Maybe make it like a weekly view instead?

**Comment by 4l3x91** [comment]
we can make a mock of this to easier get a sense of what it might look and act like

**Comment by 4l3x91** [comment]
very good idea by the way, i am proud of you @LucBen93 🧠♥🙈
